doc/marian-integration.md

```diff
@@ -93,3 +93,18 @@ ARGS=(
 </summary>
 </details>
 
+## Coding Style
+
+This repository contains C++ and JS source-files, of which C++ should adhere to
+the clang-format based style guidelines. You may configure your development
+environment to use the `.clang-format` and `.clang-format-ignore` files with
```

Comment:
Very minor. We shall mention `.clang-format` and `.clang-format-ignore` files are provided in the root directory (or if links if you prefer).